templates/home.html

```diff
@@ -19,11 +19,13 @@
         <h1 class="home-title">PERDIEM</h1>
         <h2 class="home-description">CROWDSOURCED INVESTING IN MUSIC</h2>
         {% if user.is_authenticated %}
-            <p class="home-text">Welcome {% firstof user.get_full_name user.get_username %}!</p>
+            <p class="welcome-text">Welcome {% firstof user.get_full_name user.get_username %}!</p>
         {% else %}
-            <p class="home-text"><a data-open="login-modal">Please sign in.</a></p>
+        <br>
```

Comment:
This `<br>` should be indented in a bit more. Though personally I don't think we should even use a `<br>` here at all. Using CSS with `margin-top` might be a better solution as `<br>` can have different heights in different browsers.